src/components/Checkbox/index.tsx

```diff
@@ -1,16 +1,33 @@
-import React from 'react';
+import React, { HTMLAttributes } from 'react';
 import { v4 } from 'uuid';
-import PropTypes from 'prop-types';
 import cc from 'classcat';
 
 import CheckEmptyIcon from 'lib/icons/react/CheckEmptyIcon';
 import CheckFilledIcon from 'lib/icons/react/CheckFilledIcon';
 import CheckIndeterminateIcon from 'lib/icons/react/CheckIndeterminateIcon';
 
-/**
- * @param {Object} props react props
- * @returns {ReactElement}
- */
+export interface CheckboxProps extends HTMLAttributes<HTMLElement | HTMLInputElement> {
```

Comment:
I think this should only be `HTMLInputElement` since the `rest` props are spread on an `input`.

```suggestion
export interface CheckboxProps extends HTMLAttributes<HTMLInputElement> {
```

Reply:
Done